src/ray/common/task/task_spec.cc

```diff
@@ -192,6 +192,8 @@ bool TaskSpecification::IsDriverTask() const {
   return message_->type() == TaskType::DRIVER_TASK;
 }
 
+std::string TaskSpecification::GetName() const { return message_->name(); }
```

Comment:
```suggestion
const std::string TaskSpecification::GetName() const { return message_->name(); }
```
? Any reason why the return value is not const?

Reply:
Nope, I forgot to make it const, good catch!